examples/java/README.md

```diff
@@ -0,0 +1,44 @@
+# Introduction
+
+This directory contains example app on how to uploads an object using the AWS Java SDK s3 API.
+
+# Configure credentials
+
+In your text editor create a text file  and add the following information 
```

Comment:
what is the name that this file should have?

Reply:
it is a credentials files